src/screens/lb-dapp/lb-dapp.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import { ButtonLargePrimary } from '../../components/button/button-large/button-large-primary/button-large-primary';
+import { ButtonsContainer } from '../../components/button/buttons-container/buttons-container';
+import { Divider } from '../../components/divider/divider';
+import { Icon } from '../../components/icon/icon';
+import { IconNameEnum } from '../../components/icon/icon-name.enum';
+import { LpTokenIcon } from '../../components/icon/lp-token-icon/lp-token-icon';
+import { TouchableIcon } from '../../components/icon/touchable-icon/touchable-icon';
+import { InsetSubstitute } from '../../components/inset-substitute/inset-substitute';
+import { ScreenContainer } from '../../components/screen-container/screen-container';
+import { ModalsEnum } from '../../navigator/enums/modals.enum';
+import { useNavigation } from '../../navigator/hooks/use-navigation.hook';
+import { formatSize } from '../../styles/format-size';
+import { useLbDappStyles } from './lb-dapp.styles';
+
+export const LbDapp = () => {
+  const { navigate } = useNavigation();
+  const styles = useLbDappStyles();
+
+  const onRefreshPressHandler = () => {
+    console.log('refresh pressed');
+  };
+
+  const onRemoveLiquidityHandler = () => {
+    navigate(ModalsEnum.RemoveLiquidity);
+  };
+
+  const onAddLiquidityHandler = () => {
+    console.log('add liquidity');
+  };
+
+  return (
+    <ScreenContainer>
+      <InsetSubstitute type="top" />
+      <View style={styles.lineDivider} />
+      <Divider size={formatSize(8)} />
+      <View style={styles.lbCoinContainer}>
+        <LpTokenIcon firstTokenIcon={IconNameEnum.LbTokenIcon} secondTokenIcon={IconNameEnum.LbTokenIcon} />
+        <Text style={styles.lbCoinText}>XTZ/tzBTC</Text>
+        <Divider size={formatSize(8)} />
+        <TouchableIcon name={IconNameEnum.RefreshIcon} size={formatSize(24)} onPress={onRefreshPressHandler} />
```

Comment:
This's not a refresh button, it should change the direction of chart (xtz/btc > btc/xtz)
remove it for now